docs/source/quickstart_drilldown.rst

```diff
@@ -0,0 +1,135 @@
+:orphan:
+
+.. _quickstart_drilldown:
+
+Quickstart options and troubleshooting
+============================================================
+
+..
+    Eventually, these H2s will probably all be separate articles. For now, I'm 
+    avoiding that so as not to create a bunch of super-skinny pages. 
+
+.. _quickstart_drilldown_install:
+
+Customize and troubleshoot MLflow installation
+---------------------------------------------------
+
+Python library options
+**********************
+
+Rather than the default MLflow library, you can install the following variations:
+
+.. list-table::
+   :widths: 10 20 70
+   :header-rows: 1
+
+   * - Name
+     - ``pip install`` command
+     - Description
+   * - mlflow-skinny
+     - ``pip install mlflow-skinny``
+     - Lightweight MLflow package without SQL storage, server, UI, or data science dependencies.
+   * - mlflow[extras]
+     - ``pip install mlflow[extras]``
+     - MLflow package with all dependencies needed to run various MLflow flavors. These dependencies are listed in doc:`https://github.com/mlflow/mlflow/blob/master/requirements/extra-ml-requirements.txt`
+
+
+
+Python and Mac OS X
+**********************
+
+We strongly recommend using a virtual environment manager on Macs. We always recommend using virtual environments, but they are especially important on Mac OS X because the system ``python`` version varies depending on the installation and whether you've installed the Xcode command line tools.
+
+Virtual environment managers
+****************************
+
+We release MLflow on:
+
+- PyPI (``pip install mlflow``)
+- conda-forge (``conda install -c conda-forge mlflow``)
+- CRAN (``install.packages("mlflow")``)
+- Maven Central (``mlflow-client``, ``mlflow-parent``, ``mlflow-scoring``, ``mlflow-spark``)
```

Comment:
Can we move the CRAN / Maven Central to the `R and Java` section below and then add a `Python` section for the `PyPI` and `conda-forge` elements here?

Reply:
DONE